pkg/policy/l3.go

```diff
@@ -133,6 +134,23 @@ func (m L3PolicyMap) DeepCopy() L3PolicyMap {
 	return cpy
 }
 
+func ipnetEqual(a, b net.IPNet) bool {
+	return bytes.Equal(a.IP, b.IP) && bytes.Equal(a.Mask, b.Mask)
```

Comment:
Add unit tests.

Reply:
Changed to use reflect.DeepEqual instead.